embassy-traits/src/uart.rs

```diff
@@ -10,6 +10,15 @@ pub enum Error {
 pub trait Uart {
     type ReceiveFuture<'a>: Future<Output = Result<(), Error>>;
     type SendFuture<'a>: Future<Output = Result<(), Error>>;
+    // Receive into the buffer until the buffer is full
     fn receive<'a>(&'a mut self, buf: &'a mut [u8]) -> Self::ReceiveFuture<'a>;
+    // Send the specified buffer, and return when the transmission has completed
```

Comment:
you need to use `///` to create a docstring.
```suggestion
    /// Send the specified buffer, and return when the transmission has completed
```